### ISSUE TITLE
Unable to compile in Centos Docker

### ISSUE THREAD

**Issue by kramasamy** [twitter/heron#742]
The error occurs while compiling pycrypto and it is unable to find "Python.h"

**Comment by kramasamy** [comment]
@billonahill - as a first step, you need to install docker in your mac and then do the following on the top level directory of the local heron repo

```
docker/build-artifacts.sh centos7 master ~/centos
```

**Comment by tysonnorris** [comment]
I had the same problem running docker/build-artifacts.sh centos7 master ~/centos
I took a stab at fixing it - see the PR

**Comment by aaronshan** [comment]
I also find this error when I run
```
docker/build-artifacts.sh centos7 master ~/centos
```
@kramasamy Have you resolve it?

**Comment by aaronshan** [comment]
@tysonnorris  Thank u very much. use the latest code, I have resolved this problem.

**Comment by kramasamy** [comment]
Fixed in #842 - thanks @tysonnorris
